mne/channels/channels.py

```diff
@@ -202,7 +202,8 @@ def equalize_channels(instances, copy=True, verbose=None):
                FIFF.FIFF_UNIT_T_M: 'T/m',
                FIFF.FIFF_UNIT_MOL: 'M',
                FIFF.FIFF_UNIT_NONE: 'NA',
-               FIFF.FIFF_UNIT_CEL: 'C'}
+               FIFF.FIFF_UNIT_CEL: 'C',
```

Comment:
Are we bound to a string without spaces and no special characters? Can we change this to "degrees C" or "°C"?